src/dev.Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN dotnet publish -c Release -o /app/EdFi.Tools.ApiPiblisher.Cli --no-build --n
 
 
 # Tag aspnet:8.0 alpine
-FROM mcr.microsoft.com/dotnet/aspnet@sha256:de73c1e1abd69d3ffa2658075ad4cd4edccfef37eb92ddda2c78f20173403238

```

Comment:
Regarding the error above: line 34 is unnecessary. Also, I don't think it will work now, since `AS build` was removed from the first line. I recommend restoring `AS build` on line 8 and removing line 34.

For faster local builds,  add `AS runtimeBase` one line 39, and move the `apk` command from line 55 to be right below `LABEL`. Then create the final layer from `runtimeBase`. What does this do for you? It means that the `apk` install updates will be cached in a re-usable layer locally, so that you don't need to run them every time.

```
FROM mcr.microsoft.com/dotnet/aspnet@sha256:ba398f8c6a0469436cc115bfbd278002baf4ce9423b6d8a9e904da6adc31a23d as runtimeBase
LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"

RUN apk --no-cache add unzip=~6 dos2unix=~7 bash=~5 gettext=~0 icu=~74 curl=~8

FROM runtimebase AS setup

# Now add the rest of your ENV, COPY, RUN, and ENTRYPOINT commands
```